modules/pel/peltool/peltool.py

```diff
@@ -273,14 +273,58 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
-def parsePelFromID(path: str, pelID: str, config: Config) -> None:
+def deleteAllPELs(path: str) -> None:
     """
-    Search the PEL id in the given path and prints the details.
+    Delete all files in given path
     Returns: None
     """
-    pelID=pelID.upper()
+    root = ""
+    for root, _, files in os.walk(path):
+        for file in files:
+            if not os.path.isfile(os.path.join(root, file)):
+                continue
+            os.remove(os.path.join(root, file))
+        # Only process top level directory
+        break
+
+
+def process_pelID(pelID: str) -> str:
+    """
+    Processes a string by converting it to uppercase and removing the "0X" prefix if present.
+    Returns: str: Processed pelID string.
+    """
+    pelID = pelID.upper()
     if pelID.startswith("0X"):
         pelID = pelID[2:]
+    return pelID
+
+def deletePELFromPELId(path: str, pelID: str) -> None:
+    """
+    Search the PEL id in the given path and delete it.
+    Returns: None
+    """
+    pelID = process_pelID(pelID)
+    foundID = False
+    root = ""
+    for root, _, files in os.walk(path):
+        for file in files:
+            if pelID not in file:
```

Comment:
This is looking at <commit timestamp>_<pel ID>   filenames... so it could be a little dangerous if someone doesn't pass in a full 8 characters, as it could match on a substring.  I guess you could just check to make sure the ID is the right length?